### PR TITLE
Update Example App to RN 60

### PR DESCRIPTION
**Pull request by fearmear**

### CODE REVIEW

File: ExampleApp/package.json [hunk 1]
```diff
@@ -3,20 +3,24 @@
...
-    "test": "jest"
+    "start": "react-native start",
+    "test": "jest",
+    "lint": "eslint ."
   },
   "dependencies": {
-    "@wowmaking/react-native-iron-source": "^2.8.5",
-    "react": "16.6.3",
-    "react-native": "0.58.4"
+    "@wowmaking/react-native-iron-source": "^3.0.0",
```
Comment (StanislavMayorov) [inline ExampleApp/package.json]: Let's use `extraNodeModules` in metro config to use current version from file system

Comment (fearmear) [reply ExampleApp/package.json]: Added in f1513698d9680a336db49bdc2ade360f4263ae55

Comment (StanislavMayorov) [reply ExampleApp/package.json]: `"@wowmaking/react-native-iron-source": "^3.1.0"` is it still required?

Comment (fearmear) [reply ExampleApp/package.json]: Nope.
See c9f8900711a2d504c0e9915824e18e826b5a661e where I deleted it.

**Pull request merged**